### ISSUE TITLE
UMD minimized release with a QSelect triggers an exception: invalid assignment to const 'f'

### ISSUE THREAD

**Issue by lemmel** [quasarframework/quasar#8113]
**Describe the bug**
When using the UMD minized release of Quasar 1.14.5, the following exception is triggered:
```
Uncaught TypeError: invalid assignment to const 'f'
    __setVirtualScrollSliceRange https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    __setVirtualScrollSliceRange https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    __onVirtualScrollEvt https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    s https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    setTimeout handler*o https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    __resetVirtualScroll https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    __updateMenu https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    menu https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js:6
    VueJS 4
quasar.umd.modern.min.js:6:332393
```
The exception is not triggered when the not minimized one is used.

**To Reproduce**
Use of the following code: 
```html
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">

    <title>Sandbox</title>
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons" rel="stylesheet" type="text/css">
    <link href="https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.min.css" rel="stylesheet" type="text/css">
</head>
<body>
  <script src="https://cdn.jsdelivr.net/npm/vue@^2.0.0/dist/vue.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/quasar@1.14.5/dist/quasar.umd.modern.min.js"></script>
  <div id='root'>
    <q-select v-model="model" :options="options" label="Standard"></q-select>
  </div>
  <script>
    new Vue({
      el: '#root',
      data: function () {
        return {
          model: null,
          options: [
            'Google', 'Facebook', 'Twitter', 'Apple', 'Oracle'
          ],
        }
      }
    })
</script>
</body>
</html>
```
**System**
 - Quasar UMD minified: 1.14.5
 - Vue 2.6.12
 - Firefox 83.0 and Chrome 86.0.4240.75 (64 bits)

**Additional context**
Thanks for the amazing library!

**Comment by pdanpdan** [comment]
Can you please try with the normal UMD version (minimized), not with the modern one?

**Comment by lemmel** [comment]
There is no error when the « not moderne one » is used.

(thanks)

**Comment by webnoob** [comment]
The var isn't redefined so must be a built issue: https://github.com/quasarframework/quasar/blob/64e064bc73023fba7b7b236224f0fe1a22fdf39d/ui/src/mixins/virtual-scroll.js#L405

**Comment by hawkeye64** [comment]
@webnoob @IlCallo 
Look down just a bit further
```js
                    this.prevScrollStart = _,
                    u = t,
                    p = _,
                    m = this.virtualScrollHorizontal,
                    f = f, // <-- this is the cause
```
Definitely a build bug.
It comes from this code:
```js
        setScroll(
          scrollEl,
          scrollPosition,
          this.virtualScrollHorizontal,
          rtl
        )
```
But, I think this can be fixed with:
```js
        setScroll(
          scrollEl,
          scrollPosition,
          this.virtualScrollHorizontal,
          rtl === true
        )
```
Just wondering how many more out there are like this?

**Comment by hawkeye64** [comment]
Just looked at the unminified version and it wrote it like this:
```js
          setScroll$1(
            scrollEl,
            scrollPosition,
            this.virtualScrollHorizontal,
            this.$q.lang.rtl
          );
```
I'll make a PR with it done this way: `this.$q.lang.rtl`

**Comment by hawkeye64** [comment]
Here is codepen for testing purposes: https://codepen.io/Hawkeye64/pen/ExgjVKX

**Comment by hawkeye64** [comment]
After change and builing, it's producing a `g=this.$q.lang.rtl` instead
![image](https://user-images.githubusercontent.com/10262924/100671451-bd6c6d80-331d-11eb-870f-714a7d52117b.png)

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v1.14.6. Nice work on the fix.
